kolibri/plugins/user/assets/src/vue/index.vue

```diff
@@ -1,13 +1,20 @@
 <template>
 
-  <core-base :topLevelPageName="topLevelPageName">
+  <core-base v-if="navBarNeeded" :topLevelPageName="topLevelPageName">
     <component
       slot="content"
       class="user page"
       :is="currentPage"
     />
   </core-base>
 
+  <component
```

Comment:
I was skeptical of the repeated component, but according to Vue's docs:

https://vuejs.org/v2/guide/conditional.html#Controlling-Reusable-Elements-with-key

It should be pretty efficient.